tests/test_impl_upload_download.py

```diff
@@ -248,56 +190,37 @@ def test_repeat_upload_download_dataset_using_impl(self, file_factory, did_facto
 
         # Adding files to a new dataset
         cmd = 'rucio upload --legacy --rse {0} --scope {1} --impl {2} {3} {1}:{4}'.format(rse, scope, impl, tmp_file1, tmp_dsn)
-        print(self.marker + cmd)
         exitcode, out, err = execute(cmd)
-        print(out)
-        print(err)
         # upload the files to the dataset
         cmd = 'rucio -v upload --legacy --rse {0} --scope {1} --impl {2} {3} {4} {5} {1}:{6}'.format(rse, scope, impl, tmp_file1, tmp_file2, tmp_file3, tmp_dsn)
-        print(self.marker + cmd)
         exitcode, out, err = execute(cmd)
 
         # List the files
         cmd = 'rucio list-files {0}:{1}'.format(scope, tmp_dsn)
-        print(self.marker + cmd)
         exitcode, out, err = execute(cmd)
-        print(out)
-        print(err)
-        assert exitcode == 0
```

Comment:
This commit has a functional change (it's removing an assert) while the commit message just says "[remove most print calls. Went from 329 to 69.](https://github.com/rucio/rucio/pull/7852/commits/d0bd2645075086aeae4271a1bb1684d6b173f19f)". This assert change should go in 26c459c3915f444b658c84ed8a818912e226307a